README.md

```diff
@@ -57,14 +57,15 @@ Before using `DashKit` as a react component, it must be configured.
 
 - DashKit.setSettings
 
-  Used for global DashKit settings (such as margins between widgets and default widget sizes)
+  Used for global DashKit settings (such as margins between widgets, default widget sizes and widget overlay menu)
 
   ```js
   import {DashKit} from '@yandex-cloud/dashkit';
 
   DashKit.setSettings({
     gridLayout: {margin: [8, 8]},
     isMobile: true,
+    // menu: [] as Array<MenuItem>,
```

Comment:
Why is this commented out?

Reply:
This is just an example, if we really pass an empty array, we will get an empty menu